_rules/iframe-accessible-name-cae760.md

```diff
@@ -132,10 +133,10 @@ Usage of `alt` attribute to describe content is not valid.
 
 #### Failed Example 7
 
-[Accessible name][] is empty.
+This `iframe` that is part of [sequential focus navigation][] has an empty [Accessible name][].
```

Comment:
We need a similar Passed Example (not in the accessibility tree, in sequential focus order, has an accessible name).

Reply:
I don't think that's necessary.